frontend/src/screens/learner.tsx

```diff
@@ -64,33 +66,74 @@ const LearnerDashboard = () => {
             <HighlightedStudies />
 
             <StudiesContainer />
-
+            <Affix position={{ bottom: 20, left: 20 }}>
+                <Transition transition="slide-up" mounted={isVisible}>
+                    {(transitionStyles) => (
+                        <Button
+                            onClick={() => scrollTo({ y: 0 })}
+                            style={transitionStyles}
+                            radius="xl"
+                            p={0}
+                            w={48}
+                            h={48}
+                            bg="#007bff"
+                        >
+                            <IconArrowUp size="1.5rem" color="#fff" />
+                        </Button>
+                    )}
+                </Transition>
+            </Affix>
             <Footer includeFunders />
         </div>
     )
 }
 
-export const SearchBar: FC<{search: string, setSearch: (search: string) => void}> = ({ search, setSearch }) => {
-    const isMobile = useIsMobileDevice()
+export const SearchBar: FC<{
+    search: string;
+    setSearch: (search: string) => void;
+}> = ({ search, setSearch }) => {
+    const isMobile = useIsMobileDevice();
 
     return (
         <TextInput
-            w={isMobile ? '100%' : '400px'}
-            size='lg'
+            w={isMobile ? '100%' : '600px'}
+            size='md'
             value={search}
             onChange={(event) => setSearch(event.currentTarget.value)}
-            rightSection={search.length ?
-                <IconX onClick={() => setSearch('')} style={{ cursor: 'pointer' }}/> :
-                <IconSearch />
+            rightSection={
+                search.length ? (
+                    <IconX onClick={() => setSearch('')} style={{ cursor: 'pointer' }}/>
+                ) : (
+                    <ActionIcon variant="subtle" color="#848484">
+                        <IconSearch size="1.1rem" stroke={1.5} />
+                    </ActionIcon>
+                )
             }
             placeholder="Search by study title, researcher, or topic name"
+            rightSectionWidth={40}
+            styles={(theme) => ({
+                root: {
+                    maxWidth: '400px',
+                },
+                input: {
+                    borderRadius: '50px',
+                    border: '1px solid #848484',
+                    paddingLeft: '20px',
+                    '&:focus': {
+                        borderColor: theme.colors.blue[6],
+                    },
+                },
+                rightSection: {
+                    pointerEvents: 'none',
+                },
+            })}
         />
     )
 }
 
 export const StudiesTitle: FC<{search: string, filteredStudies: ParticipantStudy[]}> = () => {
     return (
-        <Title order={2}>All Studies</Title>
+        <Title order={2}>View All Studies</Title>
```

Comment:
This container should have a left and right padding of 40px. Check the design spec